HACKING.md

```diff
@@ -0,0 +1,318 @@
+# Starcraft
```

Comment:
This file must be adapted. I understand it should contains the content of the existing `CONTRIBUTING.rst`.